### ISSUE TITLE
Semantic segmentation on COCO

### ISSUE THREAD

**Issue by luczeng** [facebookresearch/detectron2#981]
Hello, I tried the semantic semantation only branch on the coco dataset and it does not segment very well on the COCO validation dataset.

## Instructions To Reproduce the Issue:

1. Here is my code. The file semantic_R_50_FPN_1x.yaml is the official detectron2 file is from this link: https://github.com/facebookresearch/detectron2/blob/master/configs/Misc/semantic_R_50_FPN_1x.yaml. I downloaded the "R-50.pkl" weights from this link: https://dl.fbaipublicfiles.com/detectron2/ImageNetPretrained/MSRA/R-50.pkl. 

```
from detectron2.engine import DefaultPredictor
from detectron2.config import get_cfg
import cv2
from detectron2.data import MetadataCatalog,DatasetCatalog
from pathlib import Path
import matplotlib.pyplot as plt

if __name__ == '__main__':

   # Network configs
    cfg = get_cfg()
    cfg.merge_from_file("configs/COCO-PanopticSegmentation/semantic_R_50_FPN_1x.yaml")
    cfg.MODEL.WEIGHTS = str(Path("/home/luczeng/Data/Weights/R-50.pkl").absolute())

    # IO
    input_folder = "/home/luczeng/Data/Coco/panoptic_annotations_train_val2017/testimgs"

    # Tree index
    metadata = MetadataCatalog.get('coco_2017_val_100_panoptic_stuffonly')
    idx_tree = metadata.stuff_classes.index('tree')

    predictor = DefaultPredictor(cfg)

    for img_path in Path(input_folder).absolute().glob('*.jpg'):
        img = cv2.imread(str(img_path))

        res = predictor(img)
        pred = res["sem_seg"].to('cpu')

        f,(ax1,ax2) = plt.subplots(1,2)
        ax1.imshow(pred[idx_tree,:,:])
        ax2.imshow(img)
        plt.show()
```

2. I ran this script using: `python test.py`

3. Here is the results for one image taken from the coco 2017 validationset. As you can see, the trees are not segmented. I believe that the R-50 weights are only the backbone, but not the the semantic network weights. The label map I get is quite random and different at each run. Does this mean that the network's weights are initalized randomly because the R-50 weights are not complete?

![image](https://user-images.githubusercontent.com/28003333/75793493-bbb25100-5d6f-11ea-8508-2b2f65b20575.png)

## Expected behavior:

I expected the semantic FPN to work well on the coco validation set. 

## Environment:

Here is my environment:

------------------------  ------------------------------------------------------------------------------------
sys.platform              linux
Python                    3.7.5 (default, Oct 25 2019, 15:51:11) [GCC 7.3.0]
numpy                     1.17.5
detectron2                0.1 @/home/luczeng/Facade_parsing/detectron2/detectron2
detectron2 compiler       GCC 7.4
detectron2 CUDA compiler  10.2
detectron2 arch flags     sm_75
DETECTRON2_ENV_MODULE     <not set>
PyTorch                   1.3.1 @/home/luczeng/miniconda3/envs/pytorch/lib/python3.7/site-packages/torch
PyTorch debug build       False
CUDA available            True
GPU 0                     GeForce RTX 2080 Ti
CUDA_HOME                 /usr/local/cuda
NVCC                      Cuda compilation tools, release 10.2, V10.2.89
Pillow                    6.2.1
torchvision               0.4.2 @/home/luczeng/miniconda3/envs/pytorch/lib/python3.7/site-packages/torchvision
torchvision arch flags    sm_35, sm_50, sm_60, sm_70, sm_75
cv2                       4.1.2
------------------------  ------------------------------------------------------------------------------------
PyTorch built with:
  - GCC 7.3
  - Intel(R) Math Kernel Library Version 2019.0.4 Product Build 20190411 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v0.20.5 (Git Hash 0125f28c61c1f822fd48570b4c1066f96fcb9b2e)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CUDA Runtime 10.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_37,code=compute_37
  - CuDNN 7.6.3
  - Magma 2.5.1
  - Build settings: BLAS=MKL, BUILD_NAMEDTENSOR=OFF, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -fopenmp -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -O2 -fPIC -Wno-narrowing -Wall -Wextra -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Wno-stringop-overflow, DISABLE_NUMA=1, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=True, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF,

**Comment by ppwwyyxx** [comment]
As https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md said the model you're loading is an "ImageNet pretrained model" which means it's not a semantic segmentation model. We currently do not provide pre-trained semantic segmentation model.

**Comment by pvtien96** [comment]
@luczeng Did you successfully trained a semantic segmentation model? I'm interested in!
